test/pylib/manager_client.py

```diff
@@ -48,10 +48,11 @@ async def stop(self):
         """Close driver"""
         self.driver_close()
 
-    async def driver_connect(self) -> None:
+    async def driver_connect(self, server=None) -> None:
```

Comment:
Please include a type hint: `server: Optional[ServerInfo] = None`

Reply:
This is Tomek's commit from the tablets series. I wanted to keep it as is so there's no conflict later. 

I'll fix it in [the subsequent PR](https://github.com/scylladb/scylladb/pull/13438).

Reply:
done